src/components/container/App/App.js

```diff
@@ -1,8 +1,10 @@
+import HeroSection from '../../components/HeroSection/HeroSection';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
+      <HeroSection />
```

Comment:
Remove this from here please :) We will readd this later

Reply:
done!